### PR TITLE
Implement carrier crew signups using /crew slash command

### LINKED ISSUE #25: Feature: Crew signup
m.crew - toggle signup for crew role for current carrier channel
m.crews - list crews you're signed up to (in DM?)
m.crews leave all (for example) - leave all crew roles

### PR DESCRIPTION
**Pull request by cimspin**
Closes #25

### CODE REVIEW

**Comment by cimspin** [comment]
(I am super proud of this)

File: MissionAlertBot.py [hunk 1]
```diff
@@ -1089,6 +1093,50 @@ def check(message):
...
+    print(f'Crew command carrier_data: {carrier_data}')
+    if not carrier_data.channel_id:
+        # if there's no channel match, return an error
+        embed = discord.Embed(description="Try again in a carrier's channel.", color=constants.EMBED_COLOUR_ERROR)
+        await ctx.send(embed=embed, hidden=True)
+        return
+    else:
+        # we're in a carrier's channel, now we define its crew role from db
+        print(f"/crew used in channel for {carrier_data.carrier_long_name}")
+        crew_role = discord.utils.get(ctx.guild.roles, id=carrier_data.roleid)
```
Comment (gcruicks) [inline MissionAlertBot.py:1121]: This looks good, but what if the crew role does not exist? We might want to cover our bases and exit out with an error message in that case.

**Review by gcruicks — COMMENTED** [review]
I think we need to cover one more scenario here but this looks good

File: MissionAlertBot.py (hunk 1 above)
Comment (cimspin) [reply MissionAlertBot.py:1121]: Good point! Check out the latest commit for my implementation

**Review by gcruicks — APPROVED** [review]

**Pull request merged**